### PR TITLE
Solution for homework 2

### PR DESCRIPTION
**Pull request by ThinamXx**

### CODE REVIEW

**Comment by alexeygrigorev** [comment]
Can you please also add the links to the homework.md file as well as the readme one level up? Like for homework 1. Thank you!

**Comment by ThinamXx** [comment]
I have updated the new requirements.

File: course-zoomcamp/cohorts/2022/04-evaluation/homework.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+## Submit the results
+
+* Submit your results here: https://forms.gle/8TfKNRd5Jq7sGK5M9
+* You can submit your solution multiple times. In this case, only the last submission will be used 
+* If your answer doesn't match options exactly, select the closest one
+
+
+## Deadline
+
+The deadline for submitting is 5 October (Wednesday), 23:00 CEST.
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/04-evaluation/homework.md]: Let's do it for Monday

File: appendix-b-python.ipynb [hunk 2]
```diff
@@ -1515,7 +1515,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.12 ('base')",
```
Comment (alexeygrigorev) [inline appendix-b-python.ipynb]: Can you please remove this from the PR?

**Comment by alexeygrigorev** [comment]
The homework looks good. For the next one, can you please make a separate PR for homework only?

File: course-zoomcamp/cohorts/2022/04-evaluation/homework.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+```bash
+wget https://raw.githubusercontent.com/alexeygrigorev/datasets/master/AER_credit_card_data.csv
+```
+The goal of this homework is to inspect the output of different evaluation metrics by creating a classification model (target column `card`). 
+
+
+## Preparation
+
+* Create the target variable applying `LabelEncoder` on the `card` column.
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/04-evaluation/homework.md]: I'm not sure we covered this in the course. We probably need to spend some time explaining it

Comment (ThinamXx) [reply course-zoomcamp/cohorts/2022/04-evaluation/homework.md]: We can do this by mapping as well. I think I can change it.

Comment (alexeygrigorev) [reply course-zoomcamp/cohorts/2022/04-evaluation/homework.md]: That will probably be simpler indeed

File: appendix-b-python.ipynb (hunk 2 above)
Comment (alexeygrigorev) [reply appendix-b-python.ipynb]: I mean this entire file from the PR

File: course-zoomcamp/cohorts/2022/04-evaluation/homework.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+The goal of this homework is to inspect the output of different evaluation metrics by creating a classification model (target column `card`). 
+
+
+## Preparation
+
+* Create the target variable by mapping `yes` to 1 and `no` to 0. 
+* Split the dataset into 3 parts: train/validation/test with 60%/20%/20% distribution. Use `train_test_split` funciton for that with `random_state=1`.
+
+
+## Question 1:
```
Comment (SVizor42) [inline course-zoomcamp/cohorts/2022/04-evaluation/homework.md]: No need in `:`

**Pull request merged**